android/src/com/mapswithme/maps/downloader/MigrationController.java

```diff
@@ -149,7 +149,7 @@ void start(boolean keepOld)
     if (mState == State.PROGRESS)
       return;
 
-    Location loc = LocationHelper.INSTANCE.getLastLocation();
+    Location loc = LocationHelper.INSTANCE.getLastKnownLocation();
```

Comment:
Конечно важно! Если тебе нужна хоть какая-то позиция, не зависимо от времени протухания, то ты никак ее сейчас не получишь.
Или нужно по времени у позиции сделать switch и разную логику.

> On Apr 1, 2016, at 13:42, Alexander Marchuk <notifications@github.com> wrote:
> 
> In android/src/com/mapswithme/maps/downloader/MigrationController.java <https://github.com/mapsme/omim/pull/2672#discussion_r58189451>:
> 
> > @@ -149,7 +149,7 @@ void start(boolean keepOld)
> >      if (mState == State.PROGRESS)
> >        return;
> >  
> > -    Location loc = LocationHelper.INSTANCE.getLastLocation();
> > +    Location loc = LocationHelper.INSTANCE.getLastKnownLocation();
> Разве в месте использования это важно? Можно будет потом добавить методы для конкретных use-кейсов, внутри которых будет подходящие интервалы протухания. Но это уже когда появится необходимость.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2672/files/cc92bbc4b790b789ab9d08f4d3d2825564b6054a#r58189451>